worlds/dark_souls_3/Options.py

```diff
@@ -29,10 +29,56 @@ class NoEquipLoadOption(Toggle):
     display_name = "No Equip load"
 
 
-class RandomizeWeaponsLevelOption(Toggle):
-    """Enable this option to upgrade 33% ( based on the probability chance ) of the pool of weapons to a random value
-    between +1 and +5/+10"""
+class RandomizeWeaponsLevelOption(Choice):
+    """Enable this option to upgrade a percentage of the pool of weapons to a random value between the minimum and 
+    maximum levels defined. 
+    all: All weapons are eligible, both basic and epic
+    basic: Only weapons that can be upgraded to +10
+    epic: Only weapons that can be upgraded to +5"""
     display_name = "Randomize weapons level"
+    option_none = 0
+    option_all = 1
+    option_basic = 2
+    option_epic = 3
+
+
+class RandomizeWeaponsLevelPercentageOption(Range):
```

Comment:
This option is missing a display_name.
We should add display_name = "Percentage of randomized weapons" for example

Reply:
you are right! added the display name that we needed